Source/Terminals.Plugins.Web/HttpsConnectionPlugin.cs

```diff
@@ -37,12 +41,17 @@ public ProtocolOptions CreateOptions()
 
         public Image GetIcon()
         {
-            return HttpConnectionPlugin.TreeIconHttp;
+            return TreeIconHttps;

```

Comment:
Why is the icon moved to this plugin and not referencing the original icon. It looks like a bug?

Reply:
internal static readonly Image TreeIconHttps = Resources.treeIcon_http; I added this to "sync" the two plugins to be as equal as possible, could absolutely be reverted :)